tlslite/tlsrecordlayer.py

```diff
@@ -912,3 +928,45 @@ def _changeWriteState(self):
 
     def _changeReadState(self):
         self._recordLayer.changeReadState()
+
+
+    def writeHeartbeat(self, type, payload, padding_length):
```

Comment:
this method does send the hearbeat requests - how do the user application be able to know that we received response to the request we sent?